sdk/go/common/workspace/plugins_test.go

```diff
@@ -1837,19 +1824,6 @@ func TestNewPluginSpec(t *testing.T) {
 			kind:   apitype.ResourcePlugin,
 			Error:  errors.New("VERSION must be valid semver or git commit hash: abcdxyz"),
 		},
-		{
-			name:   "https prefixed git plugin",
-			source: "https://github.com/pulumi/pulumi-example",
-			kind:   apitype.ResourcePlugin,
-			ExpectedPluginSpec: PluginSpec{
-				Name:              "github.com_pulumi_pulumi-example.git",
-				Kind:              apitype.ResourcePlugin,
-				Version:           nil,
-				PluginDownloadURL: "git://github.com/pulumi/pulumi-example",
-				PluginDir:         "",
-				Checksums:         nil,
-			},
-		},
```

Comment:
These tests were removed because when we have no version we now try to reach out to the actual repo and get one.  We don't want to do that in the unit tests here, as we only use fake URLs, but the newly added integration test shows that it works.